src/sql/workbench/services/urlBrowser/browser/urlBrowserDialog.ts

```diff
@@ -0,0 +1,452 @@
+import { IAzureAccountService } from 'sql/platform/azureAccount/common/azureAccountService';
+import { azureResource } from 'azurecore';
+import { IAzureBlobService } from 'sql/platform/azureBlob/common/azureBlobService';
+import { Link } from 'vs/platform/opener/browser/link';
+import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
+import { onUnexpectedError } from 'vs/base/common/errors';
+
+export class UrlBrowserDialog extends Modal {
+
+	private _defaultBackupName: string;
+
+	private _accounts: Account[];
+	private _selectedAccount: Account;
+	private _subscriptions: azureResource.AzureResourceSubscription[];
+	private _selectedSubscription: azureResource.AzureResourceSubscription;
+	private _storageAccounts: azureResource.AzureGraphResource[];
+	private _selectedStorageAccount: azureResource.AzureGraphResource;
+	private _blobContainers: azureResource.BlobContainer[];
+	private _selectedBlobContainer: azureResource.BlobContainer;
+	private _backupFiles: azureResource.Blob[];
+
+	private _ownerUri: string;
+	private _restoreDialog: boolean;
+	private _body: HTMLElement;
+	private _accountSelectorBox: SelectBox;
+	private _tenantSelectorBox: SelectBox;
+	private _subscriptionSelectorBox: SelectBox;
+	private _storageAccountSelectorBox: SelectBox;
+	private _blobContainerSelectorBox: SelectBox;
+	private _sasInputBox: InputBox;
+	private _sasButton: Button;
+	private _backupFileInputBox: InputBox;
+	private _backupFileSelectorBox: SelectBox;
+	private _okButton: Button;
+	private _cancelButton: Button;
+	private _onOk: (value: string | PromiseLike<string>) => void;
+	public onOk: Promise<string> = new Promise(resolve => {
+		this._onOk = resolve;
+	});
+
+
+	constructor(title: string,
+		restoreDialog: boolean,
+		defaultBackupName: string,
+		@ILayoutService layoutService: ILayoutService,
+		@IThemeService themeService: IThemeService,
+		@IContextViewService private _contextViewService: IContextViewService,
+		@IAdsTelemetryService telemetryService: IAdsTelemetryService,
+		@IContextKeyService contextKeyService: IContextKeyService,
+		@IClipboardService clipboardService: IClipboardService,
+		@ILogService logService: ILogService,
+		@ITextResourcePropertiesService textResourcePropertiesService: ITextResourcePropertiesService,
+		@IAccountManagementService private _accountManagementService: IAccountManagementService,
+		@IAzureAccountService private _azureAccountService: IAzureAccountService,
+		@IAzureBlobService private _blobService: IAzureBlobService,
+		@IInstantiationService private _instantiationService: IInstantiationService
+	) {
+		super(title, TelemetryKeys.ModalDialogName.FileBrowser, telemetryService, layoutService, clipboardService, themeService, logService, textResourcePropertiesService, contextKeyService, { dialogStyle: 'flyout', hasTitleIcon: false, hasBackButton: true, hasSpinner: true });
+		this._defaultBackupName = defaultBackupName;
```

Comment:
Just declare these as private in the constructor `private _defaultBackupName: string`